cmd/plural/crypto.go

```diff
@@ -127,7 +127,7 @@ func (p *Plural) cryptoCommands() []cli.Command {
 					Usage: "a name for the key",
 				},
 			},
-			Action: requireArgs(p.handleSetupKeys, []string{"name"}),
+			Action: p.handleSetupKeys,
```

Comment:
Is name a flag there?

Reply:
yes, there is a `name` flag but the check was for the argument.